agent.go

```diff
@@ -1226,11 +1226,19 @@ func killContainerCb(pod *pod, data []byte) error {
 	}
 
 	ctr := pod.getContainer(payload.ID)
-
 	if ctr == nil {
 		return fmt.Errorf("Container %s not found, impossible to signal", payload.ID)
 	}
 
+	status, err := ctr.container.Status()
+	if err != nil {
+		return err
+	}
+	if status == libcontainer.Stopped {
+		agentLog.Info("Container not running anymore, discard signal %s", payload.Signal.String())
```

Comment:
Please can you add the container ID to this message to help with debugging.

Reply:
and if you are there, substitute 'not running' for 'is Stopped' - may as well use the official status state to help us decode the logs later.